lib/taxonomy/publishing_api_root_taxon_parser.rb

```diff
@@ -0,0 +1,27 @@
+module Taxonomy
+  module PublishingApiRootTaxonParser
```

Comment:
Perhaps this is nicer as a class to avoid the `private_class_method` thing (but I don't feel very strongly about this).